### ISSUE TITLE
Web REPL

### ISSUE THREAD

**Issue by steida** [bodil/eslint-config-cleanjs#13]
Something like https://babeljs.io/repl, but without any option. The right side should render eslint and flowtype errors. It should help people to write parts of their apps in the clean style. I feel there should be the only one pragma `/* @clean */`. It should enforce eslint airbnb syntax (it's pretty strictly) + cleanjs. And somehow /\* @flow full-coverage? */.

**Comment by JakeChampion** [comment]
This seems rather out of scope for a repository which is for linting.

**Comment by steida** [comment]
Agreed. https://github.com/cleanjs would be a better place. It looks a bit unused, and by my own experience, Github is helpful, if you kindly ask them.
